roles/installer/defaults/main.yml

```diff
@@ -38,6 +38,22 @@ tower_route_host: ''
 
 tower_hostname: '{{ deployment_type }}.example.com'
 
+# Add a nodeSelector for the AWX pods. It must match a node's labels for the pod
+# to be scheduled on that node. Specify as literal block. E.g.:
+# tower_node_selector: |
+#   disktype: ssd
+#   kubernetes.io/arch: amd64
+#   kubernetes.io/os: linux
+tower_node_selector: ''
+
+# Add node tolerations for the AWX pods. Specify as literal block. E.g.:
+# tower_tolerations: |
+#   - key: "dedicated"
+#     operator: "Equal"
+#     value: "AWX"
+#     effect: "NoSchedule"
+tower_tolerations: ''
+
```

Comment:
@athak would you be willing to add this section also on the docs at https://github.com/ansible/awx-operator#advanced-configuration? I think that will be important for our users to have a reference point on how to configure it without looking at the code.

Reply:
Added in 2a3f700.